design-editor/src/utils/utils.js

```diff
@@ -69,10 +69,12 @@ module.exports = {
 	 * @param {object} bracketsEditor
 	 * @returns {string} boolean
 	 */
-	isDemoVersion(bracketsEditor) {
+	isDemoVersion() {
+		const bracketsEditor  = this.checkGlobalContext('brackets');
 		if (bracketsEditor) {
```

Comment:
I think we don't need this 'if' here. In VSC we have mocked brackets API and assign it to 'window.brackets'

Reply:
right, I've removed it